GitUI/SplitterManager.cs

```diff
@@ -29,18 +29,18 @@ public void AddSplitter(SplitContainer splitter, string settingName, int? defaul
 
         public void RestoreSplitters()
         {
-            foreach (var splitter in _splitters)
-            {
-                splitter.RestoreFromSettings(_settings);
-            }
+            ////foreach (var splitter in _splitters)
+            ////{
+            ////    splitter.RestoreFromSettings(_settings);
+            ////}
```

Comment:
```suggestion
            /*
             Disabled as splitters have become unstable. Refer to #123456 for more details.
             
            foreach (var splitter in _splitters)
            {
                splitter.RestoreFromSettings(_settings);
            }
            */
```

Please create a new issue detailing the problem, and reference it here and everywhere else where the code is commented.

Reply:
#8745